cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/RemoteDatasetFramework.java

```diff
@@ -218,7 +220,8 @@ public void deleteAllInstances(Id.Namespace namespaceId) throws DatasetManagemen
     Id.DatasetInstance datasetInstanceId, @Nullable Map<String, String> arguments,
     ClassLoader classLoader,
     DatasetClassLoaderProvider classLoaderProvider,
-    @Nullable Iterable<? extends Id> owners) throws DatasetManagementException, IOException {
+    @Nullable Iterable<? extends Id> owners) throws DatasetManagementException, IOException,
+    ServiceUnavailableException {
```

Comment:
please remove ``ServiceUnavailableException`` if it exists after rebasing with release/3.2